pkgs/tools/package-management/nix/default.nix

```diff
@@ -110,9 +110,7 @@ let
         outputsToInstall = [ "out" "man" ];
       };
 
-      passthru = {
-        inherit fromGit;
-
+      passthru = { inherit fromGit; } // (if (!buildPerl) then { perl-bindings = nix; } else {
```

Comment:
How about `!includesPerl` rather than `buildPerl`? `buildPerl` to me sounds like an optional feature rather than a description of the package.

Reply:
indeed. That naming is far better :+1: